### ISSUE TITLE
Passing description instead of using Meta Data?

### ISSUE THREAD

**Issue by kodeine** [MurhafSousli/ng2-sharebuttons#7]
How can i pass the description since i have a dynamic text, can you please help?

Thanks

**Comment by sagarjumble** [comment]
Did you find solution for above query, I also want to pass dynamic text to the post's?

**Comment by MurhafSousli** [comment]
@kodeine Setting meta tags is not part of this package, I suggest using [`ng2-meta`](https://github.com/vinaygopinath/ng2-meta) to change the meta tags dynamically

**Comment by kodeine** [comment]
@MurhafSousli ng2-meta is not stable for angular rc1 release, i am waiting on them as well. But i think passing [text] manually would be a good enhancement. thanks for considering it.

**Comment by MurhafSousli** [comment]
@kodeine It should work now, update the package and try

**Comment by kodeine** [comment]
@MurhafSousli passing [description] on facebook button doesnt add desc in the message. Twitter it does fine.

**Comment by MurhafSousli** [comment]
Show me your code, It's working with me!
![capture](https://cloud.githubusercontent.com/assets/8130692/19622953/002acf1a-98c0-11e6-9936-73f21116a9bd.PNG)

**Comment by hackeurtalibe** [comment]
hi the description is not working on twitter

**Comment by MurhafSousli** [comment]
@hackeurtalibe twitter has only one text input (unlike others) so use the `[title]` for twitter,

**Comment by hackeurtalibe** [comment]
Thx since i need the image what i did was to put open graph tag on the serve side depending on the url 
Thx for the plugin

**Comment by hackeurtalibe** [comment]
I am wondering where the number of share are save since they are not in my db and also why the twitter count share event not working

**Comment by MurhafSousli** [comment]
@hackeurtalibe share counts comes from the service db, and not all services provide it, twitter does not.
